sources/packages/backend/apps/api/src/utilities/program-year-utils.ts

```diff
@@ -41,3 +41,20 @@ export function offeringBelongToProgramYear(
   // Check if the program start date belong to the program year.
   return isBetweenPeriod(offering.studyStartDate, programYear);
 }
+
+/**
+ * Check if a program year allows a change request.
+ * @param programYear program year entity model to be checked.
+ * @returns true if the program year allows a change request, false otherwise.
+ */
+export function allowApplicationChangeRequest(
+  programYear: Pick<ProgramYear, "programYear">,
+): boolean {
+  // Existing program years that are not allowed to submit change request.
+  // This will allow any program year from 2025-2026 and beyond to submit.
+  // Once the change request is allowed for all program years, this function
```

Comment:
Let me know if I am missing something here. During the grooming the idea that we discussed is to have the new change request process only from `2025-2026` considering the submitted appeals until `2024-2025` right?

Reply:
Yes, this method will return true to any program year `2025-2026` and on.

Reply:
yes, when all active program years are from `2025-2026` on, we can have it removed.